test/integration/dialects/mysql/query-generator.test.js

```diff
@@ -369,6 +369,11 @@ if (Support.dialectIsMySQL()) {
           arguments: ['myTable', {where: {field: {ne: null}}}],
           expectation: 'SELECT * FROM `myTable` WHERE `myTable`.`field` IS NOT NULL;',
           context: QueryGenerator
+        }, {
```

Comment:
I had no idea where this test should go, so I just searched for mysql since this bug is mysql/mariadb specific

Where do you want me to put the test exactly?